Formula/emacs-plus.rb

```diff
@@ -233,6 +235,17 @@ class EmacsPlus < Formula
     sha256 "f6cd00bcd37be03d8d83c2e590ab7d259ae37632f5954267240ba9439f799359"
   end
 
+  if build.with? "system-appearance-change"
+    unless build.with? "emacs-27-branch"
+      odie "--with-system-appearance-change is supported only when building "\
+           "with --with-emacs-27-branch"
+    end
+    patch do
+      url (PatchUrlResolver.url "system-appearance-change-emacs-27")
```

Comment:
I fixed `PatchUrlResolver` yesterday for PRs and you are the first person to actually add a new patch via PR using this resolver. 💯 Happy to see that it works.

Reply:
It works perfectly indeed! 🥳